### ISSUE TITLE
get_field returns -1 day on front-end value for Date field

### ISSUE THREAD

**Issue by artifex-media** [AdvancedCustomFields/acf#275]
It appears somehow that get_field is returning -1 day in some cases while the value correct:
![image](https://user-images.githubusercontent.com/43996366/75690672-90623000-5ca3-11ea-8d81-1d5f083f8835.png)
![image](https://user-images.githubusercontent.com/43996366/75690720-a112a600-5ca3-11ea-9293-0f83198bdaba.png)

In stead of March 2nd, it shows March 1st which is not set as the date. This does not happen with PHP's date() and DateTime.

Changing the output format won't make it budge, it will keep displaying -1 day.

I am running the latest versions of ACF & ACF Pro (as of today) + Elementor Pro

**Comment by elliotcondon** [comment]
Hi @artifex-media 

Thanks for the bug report.

Any date offset related issue will be due to the WordPress 5.3 release, which introduced a bug that affects any websites using the `date_default_timezone_set()` function.

Firstly, please read this post from WP core: https://wordpress.org/support/topic/read-this-first-wordpress-5-3-master-list/#post-12124062

According to WP core replies (on similar issue tickets), WordPress expects the default timezone to be UTC at all times and the above code will cause many "date / time" issues with WordPress 5.3.

In short, if your website is changing the default timezone via the `date_default_timezone_set()` function, you will experience a date offset issue when formatting that value via a WP function such as `date_i18n()` (which is what ACF uses).

The issue may also be related to how Elementor is formatting the saved value too. Best to check with them if they are aware of any Date offset issues with ACF.
